packages/spor-react/src/theme/components/alert-service.ts

```diff
@@ -46,7 +37,50 @@ const config = helpers.defineMultiStyleConfig({
       paddingX: "0.75rem",
       paddingTop: "0.375rem",
       paddingBottom: "0.9375rem",
-      color: "white",
+    },
+  },
+  variants: {
+    "global-deviation": {
+      container: {
+        _hover: {
+          outlineColor: "primrose",
+        },
+        _active: {
+          backgroundColor: "blonde",
+          outlineColor: "primrose",
+        },
+      },
+      outerBox: {
+        outlineColor: "primrose",
+        backgroundColor: "blonde",
+      },
+      notificationText: {
+        color: "black",
```

Comment:
Se kommentar over - vi bruker ikke `black`. Du ser forøvrig dette i `alert.ts` også - alt som ser sort ut er egentlig `darkGrey`.